eng/pipelines/publish-logs.yml

```diff
@@ -0,0 +1,18 @@
+# Build on windows desktop
+parameters:
+- name: jobName
+  type: string
+  default: ''
```

Comment:
Do AzDo parameters always need a default value, even if the parameter should be required?

Reply:
Unsure.

Reply:
The docs are ambiguous on this point or what the default value would be per type if not specified. Prefer to keep the default value explicit then because it makes it much clearer when testing for defaults in the script below.